concrete/src/Workflow/BasicWorkflow.php

```diff
@@ -141,11 +142,11 @@ public function start(WorkflowProgress $wp)
                 $ui = UserInfo::getByID($req->getRequesterUserID());
 
                 // let's get all the people who are set to be notified on entry
-                $message = t(
+                $message = array(
```

Comment:
What about moving the translatable string down a few lines, after the lines when you set the language?